### ISSUE TITLE
documentation assumes 'latest' tag on Quay exists

### ISSUE THREAD

**Issue by HarlanH** [Squarespace/pgbedrock#8]
```
Unable to find image 'quay.io/squarespace/pgbedrock:latest' locally
docker: Error response from daemon: manifest for quay.io/squarespace/pgbedrock:latest not found.
```

Using `0.1.2` at least gets the docker image to run...

**Comment by zcmarine** [comment]
Hey Harlan, thanks for pointing that out! I just addressed this via commit a67b42849c5d8472dc9a904e9adfd6f8a855e2c3. Could you try pulling again using `latest`?

**Comment by HarlanH** [comment]
Works, thanks!
